src/util/path.js

```diff
@@ -396,39 +396,192 @@
     return destinationPath;
   };
 
+  /**
+   * Calc lenght from point x1,y1 to x2,y2
```

Comment:
Length please

Reply:
i spelled it good almost everywhere, after various checks and corrections 😂